pkg/webui/constants/components.js

```diff
@@ -12,13 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import sharedMessages from '@ttn-lw/lib/shared-messages'
+
 export const dict = {
-  is: 'Identity Server',
-  as: 'Application Server',
-  ns: 'Network Server',
-  gs: 'Gateway Server',
-  js: 'Join Server',
-  edtc: 'End Device Template Converter',
-  qrg: 'QR Code Generator',
+  is: sharedMessages.componentIdentityServer,
+  as: sharedMessages.componentApplicationServer,
+  ns: sharedMessages.componentNetworkServer,
+  gs: sharedMessages.componentGatewayServer,
+  js: sharedMessages.componentJoinServer,
+  edtc: sharedMessages.componentEdtc,
+  qrg: sharedMessages.componentQrg,
```

Comment:
Either use the abbreviation for all components or none. I'd slightly prefer the abbreviations for brevity. `sharedMessages.componentNs`